lib/Config.js

```diff
@@ -39,6 +41,9 @@ export function locationConstraintAssert(locationConstraints) {
         assert(typeof locationConstraints[l].type === 'string',
             'bad config: locationConstraints[region].type is ' +
             'mandatory and must be a string');
+        assert(supportedBackends.indexOf(locationConstraints[l].type) > -1,
```

Comment:
Ah yeah, I was wondering about that. That's why we use `.indexOf` everywhere else in the code. Maybe cleaning that up could be its own separate PR